hmpps-sqs-spring-boot-autoconfigure/build.gradle.kts

```diff
@@ -42,7 +42,7 @@ dependencies {
   testImplementation("org.springframework.boot:spring-boot-starter-test")
   testImplementation("org.mockito.kotlin:mockito-kotlin:5.4.0")
   testImplementation("org.mockito:mockito-inline:5.2.0")
-  testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.10.0")
+  testImplementation("org.jetbrains.kotlinx:kotlinx-coroutines-test:1.10.1")
```

Comment:
note that I haven't bumped version of library itself since we haven't changed any of the implementation items.